gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -370,6 +370,12 @@ gst_tensor_if_set_property_supplied_value (const GValue * value,
 {
   gint i;
   gboolean is_float = FALSE;
+
+  if (!value) {
+    ml_loge ("Invalid supplied value. The value is NULL");
+    return;
+  }
```

Comment:
As far as I know, gnu89 allows mixed declarations and code. However, in my environment, this breaks the build. Please check it out.
```
$ ninja -C build
ninja: Entering directory `build'
[41/233] Compiling C object 'gst/nnstreamer/73d9e45@@nnstreamer@sha/gsttensorif.c.o'.
FAILED: gst/nnstreamer/73d9e45@@nnstreamer@sha/gsttensorif.c.o 
cc -Igst/nnstreamer/73d9e45@@nnstreamer@sha -Igst/nnstreamer -I../gst/nnstreamer -Igst/nnstreamer/./include/ -I../gst/nnstreamer/./include/ -Igst/nnstreamer/tensor_transform -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/gstreamer-1.0 -I/usr/include/orc-0.4 -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=gnu89 -g '-DVERSION="1.7.1"' '-DVERSION_MAJOR="1"' '-DVERSION_MINOR="7"' '-DVERSION_MICRO="1"' -Wwrite-strings -Wformat -Wformat-nonliteral -Wformat-security -Winit-self -Waddress -Wno-multichar -Wvla -Wpointer-arith -Wmissing-declarations -Wmissing-include-dirs -Wmissing-prototypes -Wnested-externs -Waggregate-return -Wold-style-definition -Wdeclaration-after-statement '-DNNSTREAMER_CONF_FILE="etc/nnstreamer.ini"' -DENABLE_TENSORFLOW=1 -DENABLE_TENSORFLOW_LITE=1 -DHAVE_ORC=1 -DENABLE_FLATBUF=1 -Wredundant-decls -fPIC -pthread -MD -MQ 'gst/nnstreamer/73d9e45@@nnstreamer@sha/gsttensorif.c.o' -MF 'gst/nnstreamer/73d9e45@@nnstreamer@sha/gsttensorif.c.o.d' -o 'gst/nnstreamer/73d9e45@@nnstreamer@sha/gsttensorif.c.o' -c /home/wook/Work/NNS/tmp/nnstreamer/gst/nnstreamer/tensor_if/gsttensorif.c
/home/wook/Work/NNS/tmp/nnstreamer/gst/nnstreamer/tensor_if/gsttensorif.c: In function ‘gst_tensor_if_set_property_supplied_value’:
/home/wook/Work/NNS/tmp/nnstreamer/gst/nnstreamer/tensor_if/gsttensorif.c:379:3: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
   const gchar *param = g_value_get_string (value);
   ^~~~~
cc1: all warnings being treated as errors
[50/233] Compiling C object 'gst/nnstreamer/73d9e45@@nnstreamer@sta/tensor_converter.c.o'.
ninja: build stopped: subcommand failed.
```

Reply:
it's fixed :)